README.md

```diff
@@ -422,12 +422,11 @@ specific to the sensor you are using.
 
 ### Sensor Calibration File
 
-By default, scos-sensor will use `configs/default_calibration.json` as the sensor
-calibration file. However, if`configs/sensor_calibration.json` or
-`configs/sigan_calibration.json` exist they will be used instead of the default
-calibration file. Sensor calibration files allow scos-sensor to apply a gain based
-on a laboratory calibration of the sensor and may also contain other useful
-metadata that characterizes the sensor performance. For additional
+By default, scos-sensor will use `configs/default_sensor_calibration.json` as the sensor
```

Comment:
It looks like this paragraph needs to be updated. The default file is now onboard_sensor_calibration.json. The sensor_calibration.json file will only be used if  onboard_sensor_calibration.json does not exist. Also should mention use and support of differential_calibration.json.

Reply:
Updated README in 5680801a6837a66ee28103b8b509b4eb42ac2602